blackbox/docs/general/dql/selects.rst

```diff
@@ -976,6 +976,33 @@ Some Examples::
     +---------------+-------------+
     SELECT 4 rows in set (... sec)
 
+Window Functions
```

Comment:
Not sure if we need this here as well.

If we keep it I'd also rephrase the sentence below to make it less aggregation specific